src/endpoints/http/collections/index.js

```diff
@@ -0,0 +1,18 @@
+import listCollections from './list';
+import {createCollection} from './create';
+
+export function collectionsHandler(router) {
+  //list collection info
+  router.get('/collections', function(req, res, next) {
+    var appname = req.param('appname');
+    req.log.debug({app: appname}, 'listing collections for app');
+    listCollections(req.param('appname'), req.log, req.db)
+      .then(result => {
+        req.log.trace({app: appname, result}, 'collection data listed');
+        res.json(result);
+      })
+      .catch(next);
+  });
+  //create collection
+  createCollection(router);
```

Comment:
@sedroche  I moved your collection create code to the `collections` dir to keep the code together, and made some changes. are you ok with the change?